<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvTab</class>
 <widget class="QWidget" name="EnvTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>EnvTab</string>
  </property>
  <widget class="QDoubleSpinBox" name="dirX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>灯光方向</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dirY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dirZ">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>灯光颜色</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lcR">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="value">
    <double>255.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lcB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="value">
    <double>255.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lcG">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="value">
    <double>255.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>环境光颜色</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="acR">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="value">
    <double>255.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="acB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="value">
    <double>255.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="acG">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="value">
    <double>255.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="skyboxCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>天空盒</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shadowCB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="csB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>315</y>
     <width>94</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>切换Shader</string>
   </property>
  </widget>
  <widget class="QListView" name="skyboxList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
